### ISSUE TITLE
Plugins on android device throw TypeError: Cannot read properties of undefined (reading 'postMessage')

### ISSUE THREAD

**Issue by AntGrisha** [ionic-team/capacitor-plugins#1033]
### Plugin(s)
capacitor/camera 1.3.1
capacitor/share 1.1.2
other capacitor plugins with the latest version

### Capacitor Version

```
Latest Dependencies:

  @capacitor/cli: 3.5.1
  @capacitor/core: 3.5.1
  @capacitor/android: 3.5.1
  @capacitor/ios: 3.5.1

Installed Dependencies:

  @capacitor/cli: 3.5.1
  @capacitor/core: 3.5.1
  @capacitor/android: 3.5.1
  @capacitor/ios: 3.5.1
```

### Platform(s)
android (real device - Honor 9)

### Current Behavior
When I use any capacitor plugin (let's say Share or Camera) it works perfectly in ios, however on android device all of them throw the following error:
```
VM3:313 TypeError: Cannot read properties of undefined (reading 'postMessage')
    at postToNative (VM3:310:43)
    at Object.cap.toNative (VM3:380:25)
    at VM3:461:25
```

### Expected Behavior
Plugins functionality should work as expected on android device

### Code Reproduction
```
import { Component, OnInit } from '@angular/core';
import { Share } from '@capacitor/share';

@Component({
  selector: 'app-i-share',
  templateUrl: './i-share.component.html',
  styleUrls: ['./i-share.component.scss']
})
export class IShareComponent implements OnInit {
  constructor() {}

  ngOnInit(): void {}

  share() {
    Share.share({
      title: 'See cool stuff',
      text: 'Really awesome thing you need to see right meow',
      url: 'http://ionicframework.com/',
      dialogTitle: 'Share with buddies'
    });
  }
}
```

### Other Technical Details
I am using capacitor v3 with ionic v6 and angular v12
The error cause lies here:
```
if (getPlatformId(win) === 'android') {
                // android platform
                postToNative = data => {
                    var _a;
                    try {
                        win.androidBridge.postMessage(JSON.stringify(data));
                    }
                    catch (e) {
                        // error!
                        (_a = win === null || win === void 0 ? void 0 : win.console) === null || _a === void 0 ? void 0 : _a.error(e);
                    }
                };
            }
```
It looks like androidBridge is not initialized. However I haven't touched MainActivity.java in adnroid folder, so I guess it should be fine.

Before every try I am doing Build + Sync, which works perfectly for iOS.
Could you please advise?

**Comment by Ionitron** [comment]
This issue may need more information before it can be addressed. In particular, it will need a reliable Code Reproduction that demonstrates the issue.

Please see the Contributing Guide for [how to create a Code Reproduction](https://github.com/ionic-team/capacitor/blob/HEAD/CONTRIBUTING.md#creating-a-code-reproduction).

Thanks!
Ionitron 💙

**Comment by AntGrisha** [comment]
Sorry for the delay.
I should say that the problem appeared in a project I tried to migrate from pure web implementation.
I created a new app via `ionic start --capacitor` and checked that plugins (Camera or Share for example) work fine on my android device.
After some hours of config investigation I realised, that google analytics (or tagmanager) scripts in index.html somehow affect `window.androidBridge` object and override it (probably these scripts cause errors on load and androidBridge is not initialized at all).
Will find out how to deal with analytics and ionic app, sorry for disturbing you guys!
